integration-provisioning/src/main/java/org/symphonyoss/integration/provisioning/service/UserService.java

```diff
@@ -129,6 +129,25 @@ private void createNewUser(String sessionToken, String username, String name, St
     }
   }
 
+  /**
+   * Updates user information
+   * @param sessionToken Token to access the User API.
+   * @param user User object
+   * @param name User display name
+   * @param avatar User avatar (Base64 encoded)
+   */
+  private void updateUser(String sessionToken, UserV2 user, String name, String avatar) {
+    UserAttributes userAttributes = createUserAttributes(user.getUsername(), name);
+
+    try {
+      Long userId = user.getId();
+      userApi.v1AdminUserUidUpdatePost(sessionToken, userId, userAttributes);
+      updateUserAvatar(sessionToken, userId, avatar);
+    } catch (ApiException e) {
+      throw new UpdateUserException("Failed to update user avatar", e);
```

Comment:
What if the exception was thrown from the v1AdminUserUidUpdatePost() method call?  Can updateUserAvatar() even throw ApiException?  It catches it and wraps it in UpdateUesrException.  Is UpdateUserException a subclass of ApiException?  Something seems wrong here.